mobify/astro-tutorial
# Release v0.18.0

HOW TO TEST:
- copy paste the "finished tutorial code" that is in this PR into your `app.js` and make sure the tutorial runs as expected: https://github.com/mobify/astro/pull/753 http://localhost:9000/latest/guides/understanding-astro/first-simple-app/#final-solution

Update tutorial for Astro `0.18.0` release:
- iOS to Swift 3
- Android changes to support Android plugin refactor
- JavaScript build to use Webpack
- ES6 compatible JS, turned off linter rules that continuously break when using promises so that we don't need to change all of this to `sync`/`async` (including the corresponding docs)
- Updated astro docs to reflect changes and mostly to no longer break the linter when copy/pasting code snippets. See related PR: https://github.com/mobify/astro/pull/753
